<launch>
        <node pkg="monitoring_monitors_system" type="cpu_monitoring_node" name="cpu_monitor" output="screen">
                <rosparam command="load" file="$(find monitoring_monitors_system)/config/cpumonitorconfig.yaml" />
	
	</node>
</launch>
